js/code.js

```diff
@@ -59,6 +62,75 @@ function doLogin()
 
 }
 
+function doSignUp()
+{
+    userId = 0;
+    firstName = "";
+    lastName = "";
+
+    window.location.href = "register.html";
+}
+

```

Comment:
This is the JS code that connects the "Registration API" to the front-end. @cereusware @CathyChian